trulens_eval/trulens_eval/util.py

```diff
@@ -1246,16 +1246,30 @@ def base_class(self) -> 'Class':
 
         return self
 
+    def _check_importable(self):
+        try:
+            cls = self.load()
+        except Exception as e:
+            raise ImportError(
+                f"Class {self} is not importable. "
+                "If you are defining custom feedback function implementations, make sure they can be imported by python scripts. "
+                "If you defined a function in a notebook, it will not be importable."
```

Comment:
I think we need to allow local running of notebook defined functions. So far in this; I see that the failure happens at Feedback object creation. Is it possible to only fail on deserialization back from db?

Reply:
We do allow, the important part is at https://github.com/truera/trulens/pull/295/files#diff-d2f476a66d230f8a610e2908982218970c2cc19ec1627bdef62d7744595f8eddR492

Reply:
Added test to make sure: https://github.com/truera/trulens/pull/295/files#diff-d2f476a66d230f8a610e2908982218970c2cc19ec1627bdef62d7744595f8eddR492